tests/Tests/500__jupyterhub/test-pod-affinity.robot

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Suite Setup      Begin Web Test
+Suite Teardown   Teardown
+
+*** Variables ***
+
+*** Test Cases ***
+
+Test Setting Pod Toleration Via UI
+    [Documentation]    Sets a Pod toleration via the admin UI
+    [Tags]  Sanity    Tier1
+    ...     ODS-1684
+    Menu.Navigate To Page    Settings    Cluster settings
+    Wait Until Page Contains    Notebook pod tolerations
+    Set Pod Toleration Via UI    TestToleration
+    Save Changes In Cluster Settings
+
+Verify Toleration Is Applied To Pod
+    [Documentation]    Verifies Pod spawns with toleration
+    [Tags]  Sanity    Tier1
+    ...     ODS-1685
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook With Arguments    image=s2i-minimal-notebook
+    Verify Pod Toleration    TestToleration
```

Comment:
maybe these 2 lines could be put inside `Test Setting Pod Toleration Via UI`

Reply:
That would mean having only one TC with both Polarion IDs, which I wanted to avoid.
Or are you suggesting something else?

Reply:
hmm I think ODS-1685 would complete the test in ODS-1684. So, basically, I'm proposing to have one sigle polarion ID gathering the checks from ODS-1685 and ODS-1684.

Reply:
Mh, I don't really agree. ODS-1684 is testing the admin UI and verifying that the toleration can be set (and the setting saved). 1865 then takes it a step further to verify that the toleration is actually applied.
I agree that it could be one bigger test case, but I personally prefer it this way so that if something break we know which part it is right away.

Reply:
you would know in any case which part is broken and which is right. Well anyway, we can leave as it is, don't have a strong preference here.